rskj-core/src/main/java/co/rsk/peg/BridgeSupport.java

```diff
@@ -834,11 +834,12 @@ private void saveNewUTXOs(BtcTransaction btcTx) throws IOException {
      * @throws IOException
      */
     public void releaseBtc(Transaction rskTx) throws IOException {
-        Coin pegoutValue = rskTx.getValue().toBitcoin();
+        final co.rsk.core.Coin pegoutValueInWeis = rskTx.getValue();
+        final Coin pegoutValueInSatoshis = pegoutValueInWeis.toBitcoin();
         final RskAddress senderAddress = rskTx.getSender(signatureCache);
         logger.debug(
             "[releaseBtc] Releasing {} RBTC from RSK address {} in tx {}",
```

Comment:
Forgot to update the log message here, it still says RBTC

Reply:
Done!